resources/windows/wix/project.wxs.erb

```diff
@@ -26,9 +26,6 @@
       <ComponentGroupRef Id="ProductComponentGroup" />
     </Feature>
 
-    <!-- All heat runs and WiX Fragment files containing Component elements will reference this ComponentGroup -->
-    <ComponentGroup Id="ProductComponentGroup" Directory="INSTALLDIR"/>
```

Comment:
Is there a reason this is getting removed? I don't see any explanation in the commits